frontend/public/components/resource-dropdown.tsx

```diff
@@ -53,8 +54,11 @@ const DropdownItem: React.SFC<DropdownItemProps> = ({ model, showGroup, checked
 );
 
 const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
-  const { selected, onChange, allModels, showAll, className, preferredVersions } = props;
-
+  const { selected, onChange, allModels, showAll, className, groupToVersionMap } = props;
+  const preferredVersions = groupToVersionMap?.reduce((acc, value, key) => {
+    const preferredVersion = { groupVersion: `${key}/${value}`, version: value };
+    return [preferredVersion, ...acc];
+  }, []);
```

Comment:
Don't covert this to an array. Instead just index into the map below on line 78.